server/conf/evolutions/default/50.sql

```diff
@@ -20,7 +21,7 @@ SELECT
 FROM applications
 INNER JOIN programs ON applications.program_id = programs.id
 WHERE applications.lifecycle_stage IN ('active', 'obsolete')
-GROUP BY programs.name, DATE_TRUNC('month', applications.submit_time)
+GROUP BY programs.localized_name, programs.name, DATE_TRUNC('month', applications.submit_time)
```

Comment:
issue (blocking): This database evolution has already been applied in prod, so modifying it here will have no effect on existing deployments. More info here: https://www.playframework.com/documentation/2.8.x/Evolutions

Reply:
Reverted modifications to 50.sql.